sinopia_client/test/model/SinopiaResource.spec.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Sinopia Server
+ * Sinopia Server. Supports CRUD, following loosely LDP, on Sinopia resources. Top level requires special authorization for creating a group's graph (i.e. PCC, Stanford, etc.).
+ *
+ * OpenAPI spec version: 3.0.0-beta3
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ *
+ * Swagger Codegen version: 2.4.0
+ *
+ * Do not edit the class manually.
+ *
+ */
+
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD.
+    define(['expect.js', '../../src/index'], factory);
+  } else if (typeof module === 'object' && module.exports) {
+    // CommonJS-like environments that support module.exports, like Node.
+    factory(require('expect.js'), require('../../src/index'));
+  } else {
+    // Browser globals (root is window)
+    factory(root.expect, root.SinopiaServer);
+  }
+}(this, function(expect, SinopiaServer) {
+  'use strict';
+
+  var instance;
+
+  beforeEach(function() {
+    instance = new SinopiaServer.SinopiaResource();
+  });
+
+  var getProperty = function(object, getter, property) {
+    // Use getter method if present; otherwise, get the property directly.
+    if (typeof object[getter] === 'function')
+      return object[getter]();
+    else
+      return object[property];
+  }
+
+  var setProperty = function(object, setter, property, value) {
+    // Use setter method if present; otherwise, set the property directly.
+    if (typeof object[setter] === 'function')
+      object[setter](value);
+    else
+      object[property] = value;
+  }
+
+  describe.skip('SinopiaResource', function() {
```

Comment:
like the other auto-gen model tests, i'm leaving this unimplemented and skipped for now.  gets tested indirectly by the `getResource` test.

Reply:
:ok: